src/components/TodoCreationForm.js

```diff
@@ -1,27 +1,50 @@
 import React from "react";
+import DatePicker from "react-datepicker";
+import Button from "./library/Button";
 
 export default class TodoCreationForm extends React.Component {
   state = {
-    taskName: ""
+    todoName: "",
+    dueDate: null
   }
 
   render() {
+    const disabled = !this.state.todoName || !this.state.dueDate
```

Comment:
Here too:
```javascript
const { onCreate } = this.props
const { todoName, dueDate } = this.state;
const disabled = !todoName || !dueDate;
```

Reply:
As we discussed, and to repeat my point of view on this: I thing it is easier to read short variable names rather than `this.state.dueDate`. Also, you announce at the start of the render which variables from the props and the state you will be using to render the component.